sessions/backend-update-delete/demo-start/pages/_app.js

```diff
@@ -0,0 +1,15 @@
+import { SWRConfig } from "swr";
+import GlobalStyle from "../styles";
+
+const fetcher = (url) => fetch(url).then((r) => r.json());
```

Comment:
I would use "response" or at least "res" here.

Reply:
Absolutely, an issue to fix this already exists. Thanks! :)